### PR TITLE
Process title

### LINKED ISSUE #614: Daemons process title should include the daemon name
The process title set only includes the daemon executable (eg. alignak-poller). When there are many daemons it is confusing to have several `alignak-poller` without being able to differentiate between them ... including the daemon name (eg poller-master) will make it easier.

### PR DESCRIPTION
**Pull request by mohierf**
Closes #614: replace process title (eg. broker) when the daemon get its configuration (eg. broker-master)

### CODE REVIEW

File: alignak/misc/common.py [hunk 1]
```diff
@@ -45,9 +45,9 @@
 """
 from collections import namedtuple
 try:
```
Comment (ddurieux) [inline alignak/misc/common.py]: It's an old code before some times ago, setproctitle not in requirements, so now we can import it without use try

File: alignak/daemons/brokerdaemon.py [hunk 2]
```diff
@@ -460,6 +460,8 @@ def setup_new_conf(self):  # pylint: disable=R0915,R0912
             else:
                 name = 'Unnamed broker'
             self.name = name
+            # Set my own process title
+            self.set_proctitle(self.name)
```
Comment (ddurieux) [inline alignak/daemons/brokerdaemon.py]: This is managed in self.do_daemon_init_and_start() in main function. So if you add this here, it mean the new_conf modify the satellite name? but it's defined in the ini file, so I think we could not add this.

File: alignak/misc/common.py (hunk 1 above)
Comment (mohierf) [inline alignak/misc/common.py]: Some OSes are not compatible with the setproctitle library ... it is worth keeping this exception catch

File: alignak/daemons/brokerdaemon.py (hunk 2 above)
Comment (mohierf) [inline alignak/daemons/brokerdaemon.py]: Yes but the daemon do not yet has its own name (eg. broker-master), only its type (eg. broker)

**Pull request merged**